src/main/java/gift/Entity/Wish.java

```diff
@@ -0,0 +1,33 @@
+package gift.Entity;
+
+import jakarta.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "wish")
+@IdClass(WishId.class)
+public class Wish {
+
+    @Id
+    @ManyToOne
+    @JoinColumn(name = "member_id")
+    private Member member;
+
+    @Id
+    @ManyToOne
+    @JoinColumn(name = "product_id")
+    private Product product;
+
+    public Wish() {}
```

Comment:
```suggestion
@Entity
@Table(name = "wish")
public class Wish {
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;  // 복합키 제거
    
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "member_id", nullable = false)
    private Member member;
    
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "product_id", nullable = false)
    private Product product;
    
    protected Wish() {}
```
- DDL 요구사항과 달라보이는데요. 확인해보실래요?
- 기본 생성자는 public 접근제어자를 굳이 쓰지 않아도 좋을듯하네요.

Reply:
기본 생성자와 관련된 부분을 public -> protected로 수정하였습니다.